server/src/main/java/org/apache/gravitino/server/GravitinoServer.java

```diff
@@ -75,10 +77,13 @@ public class GravitinoServer extends ResourceConfig {
 
   private final GravitinoEnv gravitinoEnv;
 
+  private final LineageService lineageService;
+
   public GravitinoServer(ServerConfig config, GravitinoEnv gravitinoEnv) {
-    serverConfig = config;
-    server = new JettyServer();
+    this.serverConfig = config;
+    this.server = new JettyServer();
     this.gravitinoEnv = gravitinoEnv;
+    this.lineageService = new LineageService();
```

Comment:
Is it better to use auxService to load in Lineage service, what do you think?

Reply:
auxService is something more heavy and not natively integrated with Gravitino, both OK for me. Is it necessary to place the lineage service in aux service?

Reply:
I'm fine with the current approach.